README.md

```diff
@@ -7,7 +7,242 @@
 A Visual Studio Code extension for [ty](https://github.com/astral-sh/ty), an extremely fast
 Python type checker and language server, written in Rust.
 
-**This project is still in development and is not ready for production use.**
+TODO: Should we change the warning to be specific to the extension?
+TODO: Add something about disabling the Pylance server to make sure ty is used for LSP features
+
+> [!WARNING]
+>
+> ty is in preview and is not ready for production use.
+>
+> We're working hard to make ty stable and feature-complete, but until then, expect to encounter bugs,
+> missing features, and fatal errors.
+
+The extension ships with `ty==0.0.0a7`.
```

Comment:
Maybe we should say "The extension includes a bundled version of ty" then link "bundled version" to the line it's configured? Otherwise, it seems easy to go stale?

Is a non-bundled version used if available?

Reply:
The release script in the Ruff VS Code extension updates this version. I think Charlie removed it from ty extension's release script but I can add it back.

https://github.com/astral-sh/ruff-vscode/blob/ab433c452de24af65c2f137084b223a1e6423482/scripts/release.py#L142-L152